zebra/rt_netlink.c

```diff
@@ -71,7 +71,7 @@
 #endif
 
 static vlanid_t filter_vlan = 0;
-
+bool skip_kernel_install;
```

Comment:
so ... this is entirely netlink-specific, right? the lack of generality concerns me - no support for the bsds, for example, and no attempt to make this new setting visible to dataplane plugins ?